app/models/apple/episode.rb

```diff
@@ -35,25 +80,26 @@ def self.publish(api, episodes)
     end
 
     def self.insert_sync_logs(episodes, results)
-      episodes_by_item_guid = episodes.map { |ep| [ep.item_guid, ep] }.to_h
+      episodes_by_guid = episodes.map { |ep| [ep.guid, ep] }.to_h
 
       results.map do |res|
         apple_id = res.dig("api_response", "val", "data", "id")
         guid = res.dig("api_response", "val", "data", "attributes", "guid")
-        ep = episodes_by_item_guid.fetch(guid)
+        ep = episodes_by_guid.fetch(guid)
 
         SyncLog
           .create(feeder_id: ep.feeder_episode.id, feeder_type: :episodes, external_id: apple_id)
       end
     end
 
-    def initialize(show:, feeder_episode:, api: nil)
+    def initialize(show:, feeder_episode:, api:, api_response: nil)
       @show = show
       @feeder_episode = feeder_episode
+      @api_response = api_response
       @api = api || Apple::Api.from_env
     end
 
-    def item_guid
+    def guid
```

Comment:
Trying to clean this up as we're presenting a unified `guid` to the join against apple's episode guid attributes (which is parsed from the RSS potentially)